src/tests/suite/daos_obj.c

```diff
@@ -921,9 +928,16 @@ io_rewritten_array_with_mixed_size(void **state)
 	assert_non_null(fbuf);
 	memset(fbuf, 0, size);
 
-	/* Disabled Pool Aggrgation */
+	/* Disabled Pool Aggregation */
 	rc = set_pool_reclaim_strategy(state, aggr_disabled);
 	assert_int_equal(rc, 0);
+	/**
+	 * set_pool_reclaim_strategy() to disable aggregation
+	 * assumes all aggregation ULTs on all servers taking
+	 * effect immediately, this may not be the case.
+	 * So adding delay so that ULTs finish the round of aggregation.
+	 */
+	sleep(10);
```

Comment:
So there is no way to validate whether aggregation is disabled on all servers (no APIs)? I feel sleep may not be the correct method. Anyway, I will let you decide...